safe/reportv4/extractors/minimum_needs.py

```diff
@@ -59,12 +60,18 @@ def minimum_needs_extractor(impact_report, component_metadata):
             if need_parameter.unit.abbreviation:
                 header = '%s [%s]' % (
                     header, need_parameter.unit.abbreviation)
+            # We don't have any layer to store minimum needs for now.
+            # So, for now just calculate it manually
             need_index = analysis_layer.fieldNameIndex(
-                need_parameter.name)
+                total_affected_field['field_name'])
```

Comment:
Correct. In the impact layer (the layer where postprocessing ran), you should find, in the future, the minimum needs.